frontend/packages/kubevirt-plugin/console-extensions.json

```diff
@@ -258,119 +286,139 @@
     "properties": {
       "model": {
         "group": "kubevirt.io",
-        "kind": "VirtualMachine"
+        "kind": "VirtualMachineInstance"
       },
-      "component": { "$codeRef": "VirtualMachinesDetailsPage.VirtualMachinesDetailsPage" }
+      "component": {
+        "$codeRef": "VirtualMachinesInstanceDetailsPage.VirtualMachinesInstanceDetailsPage"
+      }
     },
     "flags": {
       "required": ["KUBEVIRT"]
     }
   },
   {
-    "type": "console.page/route",
+    "type": "dev-console.add/action",
     "properties": {
-      "path": ["/k8s/ns/:ns/virtualmachineinstances/:name"],
-      "component": {
-        "$codeRef": "VirtualMachinesInstanceDetailsPage.VirtualMachinesInstanceDetailsPage"
-      }
+      "id": "dev-catalog-virtualization",
+      "groupId": "developer-catalog",
+      "href": "/catalog?catalogType=VmTemplate",
+      "label": "%kubevirt-plugin~Virtual Machines%",
+      "description": "%kubevirt-plugin~Create a Virtual Machine from a template%",
+      "icon": { "$codeRef": "icons.vmIconElement" }
     },
     "flags": {
       "required": ["KUBEVIRT"]
     }
   },
   {
-    "type": "console.page/resource/details",
+    "type": "console.catalog/item-type",
     "properties": {
-      "model": {
-        "group": "kubevirt.io",
-        "kind": "VirtualMachineInstance"
-      },
-      "component": {
-        "$codeRef": "VirtualMachinesInstanceDetailsPage.VirtualMachinesInstanceDetailsPage"
-      }
+      "type": "VmTemplate",
+      "title": "%kubevirt-plugin~Virtual Machines%",
+      "catalogDescription": "%kubevirt-plugin~Quickly create a virtual machine from a template.%",
+      "typeDescription": "%kubevirt-plugin~**Virtual Machines** are templates for quickly creating a virtual machine.%",
```

Comment:
Virtual Machines are templates for quickly creating a virtual machine

WTF :-) ? VMs are templates ?

can you look how we ended up with nice lyrics ? can we do something about it ?

Reply:
changed to  ... have templates for...